backend/webserver/serializers.py

```diff
@@ -10,9 +10,16 @@ class Meta:
 
 class PostSerializer(serializers.ModelSerializer):
     author = AuthorSerializer(read_only=True)
+    likesCount = serializers.SerializerMethodField('count_likes')
+    
     class Meta:
         model = Post
-        fields = ['id','author','created_at','edited_at','title','description','source','origin','unlisted','content_type','content','visibility']
+        fields = ['id','author','created_at','edited_at','title','description','source','origin','unlisted','content_type','content','visibility',"likesCount"]
+    
+    def count_likes(self, post):
+        return Like.objects.filter(post=post.id).all().count()
```

Comment:
```suggestion
        return Like.objects.filter(post=post.id).count()
```
Does it work without the `.all` ?